### PR TITLE
add requestretry_rhel_7.6.sh trim-newlines_rhel_7.6.sh build script

### PR DESCRIPTION
**Pull request by sudeepraj2**

### CODE REVIEW

File: r/requestretry/requestretry_rhel_7.6.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+then
+        nvm install v12.19.1
+fi
+
+nvm alias default v12.19.1
+
+#Installing nodejs v12.19.1 by using nvm
+curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.32.0/install.sh | bash
+source ~/.nvm/nvm.sh
+nvm install 12.19.1 && node -v
```
Comment (amitsadaphule) [inline r/requestretry/requestretry_rhel_7.6.sh:46]: node is already at the start of the script [here](https://github.com/ppc64le/build-scripts/pull/766/files#diff-cc3b2315683e35b60103853214ebac26965b0454e8033c5333aef13d566c8a61R38). I think this re-installation needs to be removed.

File: t/trim-newlines/trim-newlines_rhel_7.6.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+then
+        nvm install v12.19.1
+fi
+
+nvm alias default v12.19.1
+
+#Installing nodejs v12.19.1 by using nvm
+curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.32.0/install.sh | bash
+source ~/.nvm/nvm.sh
+nvm install 12.19.1 && node -v
```
Comment (amitsadaphule) [inline t/trim-newlines/trim-newlines_rhel_7.6.sh:46]: node is already at the start of the script [here](https://github.com/ppc64le/build-scripts/pull/766/files#diff-89554a0d975960009a6701e52efb7c4846e25f16bc131285aaf25413f495b1d6R38). I think this re-installation needs to be removed.

**Review by amitsadaphule — CHANGES_REQUESTED** [review]

File: r/requestretry/requestretry_rhel_7.6.sh (hunk 1 above)
Comment (sudeepraj2) [reply r/requestretry/requestretry_rhel_7.6.sh]: Updated the build script.

File: t/trim-newlines/trim-newlines_rhel_7.6.sh (hunk 2 above)
Comment (sudeepraj2) [reply t/trim-newlines/trim-newlines_rhel_7.6.sh]: Updated the build script.

**Review by amitsadaphule — APPROVED** [review]
LGTM

**Pull request merged**